app/Http/Controllers/Remote/RemoteController.php

```diff
@@ -107,4 +108,30 @@ public function event(Request $request)
 
         return response('', 201);
     }
+
+    public function getConfiguration(Request $request, $tokenString)
+    {
+        // Try to query the token and the node from the database
+        try {
+            $token = Models\NodeConfigurationToken::where('token', $tokenString)->firstOrFail();
```

Comment:
If i do it further down I can show a "token expired" error. I also use that in the configure script in the node and also the expired token gets deleted.